python/raft/common/cuda.pyx

```diff
@@ -47,26 +58,22 @@ cdef class Stream:
     """
 
     # NOTE:
-    # If we store _Stream directly, this always leads to the following error:
-    #   "Cannot convert Python object to '_Stream'"
+    # If we store cudaStream_t directly, this always leads to the following error:
+    #   "Cannot convert Python object to 'cudaStream_t'"
     # I was unable to find a good solution to this in reasonable time. Also,
     # since cudaStream_t is a pointer anyways, storing it as an integer should
     # be just fine (although, that certainly is ugly and hacky!).
-    cdef size_t s
 
     def __cinit__(self):
-        if self.s != 0:
-            return
```

Comment:
It's not trivial to initialize a `Stream` outside of this constructor. Is handling this case critical for something?

Reply:
If we aren't accepting a stream directly in the constructor, I can't see why this would be needed.